lib/workos/organizations/organizations.ex

```diff
@@ -0,0 +1,99 @@
+defmodule WorkOS.Organizations do 
+  import WorkOS.API 
+
+  @moduledoc """
+  The Organizations module provides resource methods for working with Organizations
+  """
+
+  @doc """
+  Create an organization
+
+  ### Parameters
+  - params (map)
+    - name (string) A unique, descriptive name for the organization
+    - allow_profiles_outside_organization (boolean - optional) Whether the Connections within this Organization should allow Profiles that do not have a domain that is set
+    - domains (array of strings - optional) List of domains that belong to the organization
+
+  ### Example
+  WorkOS.Organizations.create_organization(%{
+    domains: ["workos.com"],
+    name: "WorkOS"
+  })
+  """
+  def create_organization(params, opts \\ [])
+
+  def create_organization(params, opts)
+    when (is_map_key(params, :domains) or is_map_key(params, :allow_profiles_outside_organization)) and is_map_key(params, :name) do
+      query = process_params(params, [:name, :domains, :allow_profiles_outside_organization])
```

Comment:
nit: should this be called `body` rather than `query`, because we pass it as the body of the HTTP POST request?

Reply:
I think so too, but I think this is being named in every single module, I could rename it here tho but would also fix on other places

Reply:
I think it's fine to just fix it here. We have issues to review the other modules, so we can make a note to fix it when we do those updates 👍